packages/react-instantsearch/src/core/createInstantSearch.js

```diff
@@ -24,11 +24,16 @@ export default function createInstantSearch(defaultAlgoliaClient, root) {
       searchParameters: PropTypes.object,
       createURL: PropTypes.func,
       searchState: PropTypes.object,
+      refresh: PropTypes.bool,
```

Comment:
You could put this one required, because we will enforce the fact that the value will be `true|false` instead of `true|false|undefined|null`.